dataproc_jupyter_plugin/contollers/bigqueryController.py

```diff
@@ -25,17 +25,19 @@
     BigQueryTableInfoService,
     BigQueryTableListService,
 )
-
+from dataproc_jupyter_plugin.utils.utilities import Utilities
```

Comment:
Remove this `Utilities` package and instead import

```py
from google.cloud.jupyter_config import gcp_project
```

Reply:
Changes implemented

Reply:
This is not done yet. You still need to delete this import and remove the `utilities.py` file from the repository.

Reply:
Now deleted.